types/image.yaml

```diff
@@ -256,6 +256,17 @@
     - rfc2045
     - rfc2046
   registered: true
+- !ruby/object:MIME::Type
+  content-type: image/jxl
+  friendly:
+    en: JPEG XL Image
+  encoding: base64
+  extensions:
+  - jxl
+  xrefs:
+    person:
+    - ISO-IEC_JTC1
+  registered: true
```

Comment:
This should not be `registered: true`.

Reply:
What is the correct value for registered media types that are still waiting to be moved from the provisional registry to the main registry?

I assumed `registered: true` made most sense since that should be the case soon. Should it instead be `registrered: provisional`? Or should it be `registrered: false`?

https://www.iana.org/assignments/provisional-standard-media-types/provisional-standard-media-types.xhtml

Reply:
Right, now, `registered` is a boolean value and adding a new field requires a new version of `MIME::Types`. Let me think on this, because it feels like the answer would be to add a new field…except that might break _older_ versions of MIME::Types (unless I put it in a new file for columnar data; always an option).

Reply:
The quick way to resolve this will be to say `registered: false`. I’ve added #44 as a feature to explore.

Reply:
See the comments in the main discussion.